.gitignore

```diff
@@ -17,18 +17,15 @@
 !.vscode/extensions.json
 !.vscode/*.code-snippets
 .DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
+.env*local
 .project
 .settings/
 .eslintcache
 .stylelintcache
 lerna-debug.log
 Lerna-Profile-*.json
 
-/public/vs
+public/vs
```

Comment:
Not sure this is actually used anywhere (might have been a long time ago). Should be able to remove it